entries/v/vt.edu.json

```diff
@@ -1,13 +1,19 @@
 {
   "Virginia Tech": {
     "domain": "vt.edu",
-    "url": "https://www.vt.edu/",
     "tfa": [
       "sms",
       "call",
       "custom-software",
+      "custom-hardware",
       "u2f"
     ],
+    "custom-software": [
+      "Duo"
+    ],
+    "custom-hardware": [
+      "Duo D100 Token"
```

Comment:
Looks like Yubico OTP is also being used.
>* When prompted for a second factor, click Enter a Passcode.
>* Click to place the cursor in the text box.
> * Tap your YubiKey.

On the other hand, I don't see any mention of U2F :thinking:

Reply:
I'll fix that